### PR TITLE
Reorganized the environments page to flow better.

### PR DESCRIPTION
**Pull request by zechdc**
- Added details about how to set the environment for an application
- Added details about setting the environment for Oil
- I updated the information regarding server environment variables as stevewest suggested. Let me know if this is still inaccurate. 

Signed-off-by: Zechariah Campbell

### CODE REVIEW

File: general/environments.html [hunk 1]
```diff
@@ -77,74 +110,77 @@ <h3 id="possible_envs">Environments list</h3>
...
-        stuff.php
-      production/
-        stuff.php</code></pre>
+			<h3 id="setting_env">Set Your Environment</h3>
 
 			<p>
-				When your environment is set to <kbd>\Fuel::DEVELOPMENT</kbd>, the settings from <em>stuff.php</em> are merged with <em>development/stuff.php</em>.
-				If the environment setting is set to any other than <kbd>\Fuel::DEVELOPMENT</kbd>, the extra config file is not loaded. The same goes
-				for any other environment setting you might use.
+				There are two ways to set your environment in FuelPHP. The third option below shows you how to set your environment when using FuelPHP's Oil. Oil does not use the environment you set in the first two options below, it has to be set separately, every time you use Oil.
```
Comment (emlynwest) [inline general/environments.html]: "there are two ways" then you say "the third", so there are three options? This might need some clarification.

**Comment by emlynwest** [comment]
Much cleaner, thanks!

File: general/environments.html (hunk 1 above)
Comment (zechdc) [inline general/environments.html]: Ok, I changed that. I committed the change. It looks like it was automatically added to this pull request. Is that correct or am I reading github wrong?

**Comment by zechdc** [comment]
Looks like I messed it up again. :( One more time. Here we go.

**Comment by zechdc** [comment]
Ok, I believe I got it fixed now.

File: general/environments.html (hunk 1 above)
Comment (emlynwest) [inline general/environments.html]: The green lines are things you have added or changed, then the red are things that have been removed.
"There are two ways to set your environment in FuelPHP. The third option below shows you how to set..." could be a little confusing, "If there are only two ways then where has the 3rd suddenly come from?"

Comment (zechdc) [inline general/environments.html]: I agree. I believe it has been corrected in the latest commit: 3854e9f

**Comment by emlynwest** [comment]
Looks much better, if @WanWizard does not have any issues I am happy for this to be merged in.

**Comment by WanWizard** [comment]
Nope.

**Pull request merged**